test/stdlib/FloatingPoint.swift.gyb

```diff
@@ -118,17 +117,17 @@ FloatingPoint.test("BinaryFloatingPoint/genericIntegerConversion") {
 FloatingPoint.test("BinaryFloatingPoint/genericFloatingPointConversion") {
   func convert<
     T: BinaryFloatingPoint, U: BinaryFloatingPoint
-  >(exactly value: T, to: U.Type) -> U { U(exactly: value) }
+  >(exactly value: T, to: U.Type) -> U? { U(exactly: value) }
```

Comment:
I'm confused what happened here. Did it always not compile?

Reply:
The entire test file was disabled 2 years ago (apple/swift#23423), probably due to the `isQuietNaN` test failures.

The `genericFloatingPointConversion` test was updated 5 months ago (apple/swift#33910), but it never compiled.